src/shared/telemetry/telemetryUtils.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as vscode from 'vscode'
+import { ext } from '../extensionGlobals'
+import { Datum, ResultWithTelemetry } from './telemetryEvent'
+
+export function defaultMetricDatum(name: string): Datum {
+    return {
+        name: name,
+        unit: 'Count',
+        value: 1
+    }
+}
+
+export function registerCommand<T>({
+    command,
+    callback,
+    thisArg,
+    register = vscode.commands.registerCommand
+}: {
+    command: string,
+    thisArg?: any,
+    register?: typeof vscode.commands.registerCommand
```

Comment:
Is this okay without a trailing comma? Also, please order these args the same as the block above (line 21-24)

Reply:
fixed.